### PR TITLE
Split multiple resource yaml files

### PR DESCRIPTION
**Pull request by zxDiscovery**
Helm chart recommend the best way is one yaml file only include one resource. So split multiple resource in one yaml file to multiple yaml files with one resource.

### CODE REVIEW

File: install/kubernetes/helm/istio/charts/pilot/templates/gateway/autogeneratedingress.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: istio-autogenerated-k8s-ingress
```
Comment (gyliu513) [inline install/kubernetes/helm/istio/charts/pilot/templates/gateway/autogeneratedingress.yaml]: istio-autogenerated-k8s-ingress.yaml

File: install/kubernetes/helm/istio/charts/pilot/templates/gateway/meshexpansiongateway.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{{- if .Values.global.meshExpansion }}
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: meshexpansion-gateway
```
Comment (gyliu513) [inline install/kubernetes/helm/istio/charts/pilot/templates/gateway/meshexpansiongateway.yaml]: meshexpansion-gateway.yaml

File: install/kubernetes/helm/istio/charts/pilot/templates/gateway/meshexpansionilbgateway.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{{- if .Values.global.meshExpansionILB }}
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: meshexpansion-ilb-gateway
```
Comment (gyliu513) [inline install/kubernetes/helm/istio/charts/pilot/templates/gateway/meshexpansionilbgateway.yaml]: meshexpansion-ilb-gateway.yaml

File: install/kubernetes/helm/istio/charts/security/templates/cleanup/ClusterRole.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: istio-cleanup-secrets-{{ .Release.Namespace }}
```
Comment (gyliu513) [inline install/kubernetes/helm/istio/charts/security/templates/cleanup/ClusterRole.yaml]: How about use folder name as `cleanup-secrets` and change the file name to `clean-clusterrole.yaml`

ditto for others

File: install/kubernetes/helm/istio/charts/pilot/templates/gateway/meshexpansionilbgateway.yaml [hunk 5]
```diff
@@ -0,0 +1,23 @@
+{{- if .Values.global.meshExpansionILB }}
```
Comment (sdake) [inline install/kubernetes/helm/istio/charts/pilot/templates/gateway/meshexpansionilbgateway.yaml]: this file is deprecated and should be removed.

**Comment by zxDiscovery** [comment]
Close this issue. We don't need to split multiple resource yaml files at present

**Pull request closed without merging**